README.md

```diff
@@ -1,62 +1,48 @@
-4. Automatically delete head branches.
-
-#### Branch protection rules
-Protect the default branch using the following rules:
-1. Require a pull request before merging. Require approvals.
-2. Require status checks to pass before merging. Require branches to be up to date before merging.
-3. Specify status checks that are required.
-4. Require conversation resolution before merging.
-5. Do not allow bypassing the above settings.
-
-#### Actions/General
-1. Allow all actions and reusable workflows.
-2. Allow GitHub Actions to create and approve pull requests (if dependabot is used and it's configured to automatically
-merge pull requests).
-
-### Specification
-The `spec.go` file provides a programmatic representation of the configuration options.
-
-### Configuration
-This template provides two types of "configs":
-* general configuration (that applies to both, sources and destinations)
-* and source/destination specific configs.
-
-All are defined in `config.go` and are represented by separate types. General configs should be added to `pinecone.Config`,
-whereas any source or destination specific configs should be added to `pinecone.SourceConfig` and 
-`pinecone.DestinationConfig` respectively.
+# Conduit destination connector for Pinecone
+
+The Pinecone connector is one of [Conduit](https://github.com/ConduitIO/conduit) standalone plugins. It provides a destination connector for [Pinecone](https://www.Pinecone.io/).
+
+It uses the [gRPC go Pinecone client](github.com/Pinecone-io/go-Pinecone) to connect to Pinecone.
+
+## Which OpenCDC data format does the destination connector accept?
+
+The destination connector expects an OpenCDC formatted record with the following fields
+
+| Field                   | Description                                                                             |
+|-------------------------|-----------------------------------------------------------------------------------------|
+| `record.Operation`      | which conduit operation does the record do.                                             |
+| `record.Metadata`       | a string to string map representing the Pinecone vector metadata.                       |
+| `record.Key`            | the vector id that is being updated.                                                    |
+| `record.Payload.Before` | <discarded>                                                                                 |
+| `record.Payload.After`  | the vector body, in json format             | 
+| `(json) record.Payload.After.values`  | an array of float32 representing the vector values              | 
+| `(json) record.Payload.After.sparse_values`  | **(optional)** the sparse vector values               | 
+| `(json) record.Payload.After.sparse_values.indices`  | an array of uint32 representing the sparse vector indices              | 
+| `(json) record.Payload.After.sparse_values.values`  | an array of float32 representing the sparse vector values               | 
```

Comment:
It would be good to clarify if `.Payload.After` is expected to be a RawData field with a JSON in it, or can it be a StructuredData instance as well.